src/platforms/node/guides/aws-lambda/index.mdx

```diff
@@ -80,6 +80,36 @@ exports.handler = Sentry.AWSLambda.wrapHandler(yourHandler, {
 });
 ```
 
+
+## `Promise.allSettled` Returned by the Handler
+_(New in version ??)_
+
+By default, Sentry captures errors raised by your handler.
+However, your handler might return a `Promise.allSettled` result.
+In this case, even if one of the messages has failed, Sentry won't capture any exception.
+
+The `captureAllSettledReasons` (default: `false`) option captures all promise rejected results:
+
+```javascript {tabTitle:promiseSettledDefault}
+exports.handler = Sentry.AWSLambda.wrapHandler(() => {
+  return Promise.allSettled([
+        Promise.rejected(new Error('first')),
+        Promise.rejected(new Error('second'))
+  ]);
+});
+// Nothing captured
+```
```

Comment:
This is the default behavior, I don't think adding this code snippet provides much value.

Reply:
remove exemple
1e66df5